olive/systems/docker/docker_system.py

```diff
@@ -56,12 +57,17 @@ def __init__(
                 if local_docker_config.build_context_path and local_docker_config.dockerfile:
                     build_context_path = local_docker_config.build_context_path
                     logger.info(f"Building image from Dockerfile {build_context_path}/{local_docker_config.dockerfile}")
-                    self.image = self.docker_client.images.build(
-                        path=build_context_path,
-                        dockerfile=local_docker_config.dockerfile,
-                        tag=local_docker_config.image_name,
-                        buildargs=local_docker_config.build_args,
-                    )[0]
+                    try:
+                        self.image, build_logs = self.docker_client.images.build(
+                            path=build_context_path,
+                            dockerfile=local_docker_config.dockerfile,
+                            tag=local_docker_config.image_name,
+                            buildargs=local_docker_config.build_args,
+                        )
+                        _print_docker_logs(build_logs, logging.INFO)
```

Comment:
Not sure how large the logs are but I think Debug might be better? 
info might be too verbose since it’s also the default verbosity for olive workflows.

Reply:
will change to DEBUG